### PR TITLE
Customer anonymisation

### PR DESCRIPTION
**Pull request by Oksydan**
I added a new feature, customer's lastname anonymisation. Related issue here https://github.com/PrestaShop/PrestaShop/issues/16339.

### CODE REVIEW

**Comment by PierreRambaud** [comment]
I don't understand how you add commits in your pull request :thinking: 
Could you rebase your branch please?

**Comment by Oksydan** [comment]
I am sorry about trivial question but is there way of doing rebase without creating another pull request?

**Comment by Hydrog3n** [comment]
@Oksydan Hello, you can rebase on your branch and use push force to update your branch on github and this PR will be automatically updated :)

**Comment by Oksydan** [comment]
I did few mistakes during rebase I have no idea is current state of this branch ok. Sorry about this mess. It is my first rebase.

**Comment by YanisseZ** [comment]
Thank You ! @Oksydan 

[url=https://www.noelshack.com/2020-10-5-1583495751-capture.png][img]https://image.noelshack.com/minis/2020/10/5/1583495751-capture.png[/img][/url]

**Comment by sarahdib** [comment]
@Junebyun 
This new feature add two sentences. Could you confirm that the new wording is ok for you ? Thank you :) 
Capture d’écran 2020-06-24 à 09 19 52

**Comment by Junebyun** [comment]
@sarahdib I would change the wording to: "_Make anonymous the customer's last name._"

**Comment by sarahdib** [comment]
@Junebyun thanks 

@Oksydan could you change the wording by this one : "_Make anonymous the customer's last name_." instead Anonymous the customer's last name.

**Comment by Oksydan** [comment]
@sarahdib changed

File: productcomments.php [hunk 1]
```diff
@@ -339,6 +342,25 @@ public function renderConfigForm()
...
+                        'values' => array(
+                            array(
+                                'id' => 'active_on',
+                                'value' => 1,
+                                'label' => $this->trans('Enabled', [], 'Modules.Productcomments.Admin'),
+                            ),
+                            array(
+                                'id' => 'active_off',
+                                'value' => 0,
+                                'label' => $this->trans('Disabled', [], 'Modules.Productcomments.Admin'),
```
Comment (Junebyun) [inline productcomments.php:360]:
```suggestion
                        'label' => $this->trans('Anonymize the user\'s last name.', [], 'Modules.Productcomments.Admin'),
                        'name' => 'PRODUCT_COMMENTS_ANONYMISATION',
                        'desc' => $this->trans('Display only initials, e.g. John D.', [], 'Modules.Productcomments.Admin'),
                        'values' => array(
                            array(
                                'id' => 'active_on',
                                'value' => 1,
                                'label' => $this->trans('YES', [], 'Modules.Productcomments.Admin'),
                            ),
                            array(
                                'id' => 'active_off',
                                'value' => 0,
                                'label' => $this->trans('NO', [], 'Modules.Productcomments.Admin'),
```

Comment (Oksydan) [reply productcomments.php]: @Junebyun changed

**Comment by Progi1984** [comment]
Thanks @Oksydan & @sarahdib

**Pull request merged**